src/client.cpp

```diff
@@ -393,7 +393,14 @@ status_t Client::Impl::uploadBackground(id_t accountId, id_t instrumentId,
 //    -F "data=@/path/to/image.png;type=image/png"    cs->addFormField(kStrKey, kStrFLIPBOOK);
     cs->addFormField(kStrKey, kStrBACKGROUND);
     cs->addFormField(kStrTimestamp, toIsoTimeString(timestamp));
-    cs->addFormFile(kStrData, imageFile, mimeTypeFromFilePath(imageFile));
+
+    if (payload.data)
+        cs->addFormFile(kStrData, payload.data, payload.dataSize, payload.mimeType.c_str());
```

Comment:
If it is sent as binary that does not go via Base64 that good. 
How you know it is sent like it?